solana/ts/sdk/ntt.ts

```diff
@@ -800,14 +800,20 @@ export class SolanaNtt<N extends Network, C extends SolanaChains>
   }
 
   async getIsApproved(attestation: Ntt.Attestation): Promise<boolean> {
-    const digest = (attestation as WormholeNttTransceiver.VAA).hash;
-    const vaaAddress = utils.derivePostedVaaKey(
-      this.core.address,
-      Buffer.from(digest)
-    );
-
-    const info = await this.connection.getAccountInfo(vaaAddress);
-    return info !== null;
+    if (attestation.payloadName !== "WormholeTransfer") return false;
```

Comment:
maybe better to throw here? this would indicate a bug in the calling code, right?

Reply:
i don't see any harm in throwing, i was just following the pattern used elsewhere but i'll change it